### PR TITLE
Md 4755

### PR DESCRIPTION
**Pull request by atul2103**
1. File Selector Component
2. Basic Progress Bar Component

### CODE REVIEW

File: src/components/FileSelector/index.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import { Avatar } from '@mui/material';
```
Comment (abhijit-parspec) [inline src/components/FileSelector/index.tsx:1]: follow the below order:

React Imports
ThiredParty Imports

Parspec Library Imports

Relative Imports

File: src/components/FileSelector/index.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    onUpload,
+    uploadFile = true,
+    url,
+    error,
+    helperText,
+    onSelect
+}: {
+    maxFiles?: number;
+    acceptedFormats?: string[];
+    onUpload?: (args: {}[]) => void;
```
Comment (abhijit-parspec) [inline src/components/FileSelector/index.tsx:22]: do we know the array element object's shape??

File: src/components/FileSelector/index.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    uploadFile?: boolean;
+    url: string;
+    error?: string;
+    helperText?: string;
+    onSelect?: (args: {}[]) => void;
```
Comment (abhijit-parspec) [inline src/components/FileSelector/index.tsx:27]: same as above

**Review by abhijit-parspec — CHANGES_REQUESTED** [review]

File: src/components/FileSelector/index.tsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    url,
+    error,
+    helperText,
+    onSelect
+}: {
+    maxFiles?: number;
+    acceptedFormats?: string[];
+    onUpload?: (args: {}[]) => void;
+    uploadFile?: boolean;
+    url: string;
```
[the remark this answers is not in the document]
Comment (atul2103) [reply src/components/FileSelector/index.tsx:24]: we discussed that we will pass url from the parent component. That is why it is kept mandatory.

**Review by abhijit-parspec — APPROVED** [review]

**Pull request merged**